src/pages/Home.css

```diff
@@ -76,6 +83,7 @@ nav {
     display: flex;
     align-items: center;
     padding: 0.5rem;
+    margin-left: 10vh;
```

Comment:
you should use no more than 4rem for margin: if you need more, you're facing a layout problem, not spacing. try flexbox/grid on the container instead

Reply:
Fixed